butane/build.rs

```diff
@@ -1,16 +1,22 @@
 fn main() {
-    // This build.rs is only needed to help the tests
-    if std::env::var("CARGO_CFG_TEST").is_err() {
-        return;
+    println!("cargo:rerun-if-changed=tests/");
+
+    // Note that #[cfg(test)] and CARGO_CFG_TEST do not work in build.rs
+    // See https://github.com/rust-lang/cargo/issues/4789
+    #[cfg(feature = "_auto_delete_dot_butane")]
+    {
+        clean_dot_butane();
     }
-    println!("cargo:rerun-if-changed=tests");
+}
```

Comment:
We could also reduce how often this runs by disabling it when the `PROFILE` = `release`
https://stackoverflow.com/questions/59277384/is-it-possible-to-invoke-a-build-rs-only-for-release-mode

The env will be quicker to check than looking for existence of a directory on the filesystem.